src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let sum = 0;
+
+  if (days < 3) {
+    sum = days * 40;
+  }
+
+  if (days >= 3 && days < 7) {
+    sum = days * 40 - 20;
+  }
+
+  if (days >= 7) {
+    sum = days * 40 - 50;
+  }
```

Comment:
what is `40`, `20` and `50`? add some meaningful constants; 

and why do we need to assign all values to `sum`? we can just return it at once

```suggestion
  if (days < 3) {
    return days * 40;
  }

  if (days >= 3 && days < 7) {
    return days * 40 - 20;
  }

  if (days >= 7) {
    return days * 40 - 50;
  }
```